apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/BeneficiaryTransformerV2.java

```diff
@@ -354,16 +356,15 @@ private static void addUnhashedIdentifier(
       Period mbiPeriod) {
 
     if (mbiPeriod != null && (mbiPeriod.hasStart() || mbiPeriod.hasEnd())) {
-
       patient
           .addIdentifier()
           .setValue(value)
           .setSystem(system)
           .setPeriod(mbiPeriod)
           .getType()
           .addCoding()
-          .setCode("MB")
-          .setDisplay("Member Number")
+          .setCode("MC")
```

Comment:
You are missing a `setSystem` on the type coding.

![image](https://user-images.githubusercontent.com/29063436/115895855-798b1980-a420-11eb-8a06-38e111169075.png)

Should be `"system": "http://terminology.hl7.org/CodeSystem/v2-0203"`

Reply:
Ditto the system now appearing